packages/account-v2/src/containers/TradingAssessmentForm/RiskToleranceWarningModal.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { StandaloneTriangleExclamationFillIcon } from '@deriv/quill-icons';
+import { Button, Modal, Text } from '@deriv-com/ui';
+import { ACCOUNT_MODAL_REF } from '../../constants';
+import { riskToleranceWarningModalConfig } from '../../modules/src/TradingAssessmentConfig';
+
+type TRiskToleranceWarningModal = {
+    handleSubmit: () => void;
+    isModalOpen: boolean;
+};
+
+export const RiskToleranceWarningModal = ({ handleSubmit, isModalOpen }: TRiskToleranceWarningModal) => {
+    Modal.setAppElement(ACCOUNT_MODAL_REF);
+    return (
+        <Modal className='md:w-[440px] sm:w-[312px]' isOpen={isModalOpen}>
```

Comment:
If this width is small, shouldnt this be a Dialog ?

Reply:
IMO, the content is enough to decide the width. We should not hardcode width like this. Worst case add padding to the content

Reply:
We have to fix the width here since the strings are long.